packages/react-variant-view/src/component/functionalPrediction/FunctionalPrediction.tsx

```diff
@@ -31,9 +31,7 @@ class FunctionalPrediction extends React.Component<IFunctionalPredictionProps> {
         genomeNexusData: VariantAnnotation | undefined
     ): IFunctionalImpactData {
         const mutationAssessor =
-            genomeNexusData &&
-            genomeNexusData.mutation_assessor &&
-            genomeNexusData.mutation_assessor.annotation;
+            genomeNexusData && genomeNexusData.mutation_assessor;
```

Comment:
we can probably just do this now
```suggestion
            genomeNexusData?.mutation_assessor;
```